packages/dependency-operator-filter/contracts/mock/UnRegisteredToken.sol

```diff
@@ -0,0 +1,106 @@
+//SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+import {ERC1155Upgradeable} from "@openzeppelin/contracts-upgradeable/token/ERC1155/ERC1155Upgradeable.sol";
+import {Initializable} from "@openzeppelin/contracts-upgradeable/proxy/utils/Initializable.sol";
+import {OperatorFiltererUpgradeable} from "../OperatorFiltererUpgradeable.sol";
+import {IOperatorFilterRegistry} from "../interfaces/IOperatorFilterRegistry.sol";
+
+contract UnRegisteredToken is ERC1155Upgradeable, OperatorFiltererUpgradeable {
+    function initialize(
+        string memory uri_,
+        address subscription,
+        bool subscribe
+    ) external initializer() {
+        __ERC1155_init(uri_);
+        __OperatorFilterer_init(subscription, subscribe);
+    }
+
+    /// @notice sets registry
+    /// @param registry address of registry
+    function setRegistry(address registry) external {
+        operatorFilterRegistry = IOperatorFilterRegistry(registry);
+    }
```

Comment:
Qu: we have no way to change the registry that Asset.sol uses, is that correct? I am wondering whether we need to introduce an admin setRegistry function in Asset, in case we need to update the registry address. What do you think?